README.md

```diff
@@ -1,2 +1,343 @@
+<<<<<<< HEAD
 # amdf-documentation
 Advanced Material Data Format Documentation
+=======
+# Advanced Material Data Format
+*Version: 2.0*
+
+## Content
+
+1. [Description](#description)
```

Comment:
Use 1. for all items, GitHub will render numbers accordingly

Reply:
Ok, so you mean:
1. \[Description\](\#description)
1. \[Material Types\](\#material-types)
1. \[Color Spaces\](\#color-spaces)
...